doc/method/sources.md

```diff
@@ -0,0 +1,104 @@
+### Taxonomy sources
+
+*NCBI Taxonomy.*  The first objective of the Open Tree taxonomy is to
+align OTUs across phylogenetic studies.  This need is largely met by
+using the NCBI taxonomy, since (1) most phylogenies are molecular, (2)
+publishers require molecular sequences used in studies to be deposited
+in Genbank, (3) every Genbank deposit is annotated with a taxon from
+NCBI taxonomy.  The NCBI taxonomy therefore forms the nucleus of OTT.
+The particular version of NCBI taxonomy used in the OTT assembly described below (OTT 2.10) was downloaded on
+[date], but we have retrieved a fresh version with every OTT build.
+
+*GBIF backbone taxonomy.*  The NCBI taxonomy has only [NNN] species
+records with standard binomial names.  In order for Open Tree to
+improve its coverage of the tree of life (the second objective), we
+needed a second source taxonomy providing species going beyond NCBI's
+[NNN] and reaching toward the [MMM] species that are described in the
+literature [citation needed].
+
+An initial attempt to use a global taxonomy in progress from another
```

Comment:
I guess not. I was thinking (1) it is part of the history within the open tree project ('preottol'), and (2) it helps draw out some of the concerns that guided later development. It would be nice to say (implicitly) that the preottol approach was not good because it was used bad sources and was opaque (and AFAIK not updateable, but I could be wrong since it's so hard to know anything about it).